components/gitops/staging/base/authentication/gitops-rolebindings.yaml

```diff
@@ -52,5 +52,5 @@ subjects:
     name: GitOps Service Team
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
+  kind: Role
   name: gitops-namespaces-delete-pods-access
```

Comment:
So the team will have read access as described above, and be able to delete pods in the  gitops namespaces for staging.